File: mario.c
#include <stdio.h>
#include <stdlib.h>
#include <cs50.h>

int main()
{
        int size;
     do {
         printf("height: ");
         size = GetInt();
    } while ( (size > 23) || (size < 0));
    
        
    for (int i = 1; i <= size; i++)
    {
        for (int y = i;  y < size; y++)
        {
                printf(" ");
        }
        for (int j = 0; j < i; j++)
        {
            printf("#");
        }
        printf("#\n");
       
    }
        return 0;
}

File: pset6/mario.py
import cs50

def stairs(n):        
    for i in range(0, n):
        for y in range(n -1, i, -1):
            print(" ", end="")
        for j in range(i):
            print("#", end="")
        print("##")    

n = 0
while True:
    print("height: ", end="")
    n = cs50.get_int()
    if 1 <= n <= 23:
        stairs(n)   
        break